example_app/web/templates/page/index.html.eex

```diff
@@ -37,3 +54,9 @@
     </ul>
   </div>
 </div>
+
+<script type='text/javascript'>
+  $('.logout-link').click(function() {
+    $.ajax({url:"/logout", method: "DELETE"}).done(function() { window.location.reload(); });
+  });
```

Comment:
`<%= link "Logout", to: logout_path(@conn, :delete), method: :delete %>`  looks cleaner, but my understanding is that the `logout_path` action just returns JSON and not will trigger a redirect after the logout is successful.   We need to specify a callback to reload the page once the logout is done, see `.done(function() { window.location.reload(); });` above.  I don't believe there is way to do this using the `link` function.  I'm open to feedback and other ideas though